### PR TITLE
Update GameAssist to version 0.1.4.7 and enhance documentation

### PR DESCRIPTION
**Pull request by Mord-Eagle**
This updates the existing One-Click GameAssist package from v0.1.1.2 to v0.1.4.7.

The intermediate v0.1.2-v0.1.4.6 project releases were developed and tested outside the One-Click repository. They remain documented in the project changelog and are available at https://github.com/Mord-Eagle/GameAssist but are not listed as previousversions because matching published version folders do not exist. The existing published rollback versions 0.1.1.2, 0.1.1.1, and 0.1.1.0 are preserved.

Package changes
adds GameAssist/0.1.4.7/GameAssist.js;
updates GameAssist/script.json to v0.1.4.7 using the required GameAssist.js filename and property-based modifies declarations;
updates the One-Click description, dependencies, named conflicts, commands, setup instructions, and troubleshooting documentation;
replaces the older package README with the current v0.1.4.7 documentation.
Release highlights
clearer !ga-status system health and troubleshooting panels;
configuration self-healing and versioned configuration snapshots;
improved CritFumble help and natural-1 workflow;
exact built-in and custom concentration-marker recognition;
NPC death tracking with Campaign, Chapter, Section, and Session handouts;
NPC death audit, report writing, hierarchical clearing, and curated Arc records;
NPCHPRoller initialization protection that prevents false death/revival history;
verified TokenMod marker requests through its documented --api-as path;
optional StatusInfo detection and observer-path diagnostics.
Requirements and compatibility
D&D 5E (2014) character sheet;
TokenMod 0.8.88 supported baseline for NPCManager and ConcentrationTracker marker changes;
StatusInfo 0.3.11 optional for condition descriptions and menus;
seven campaign rollable tables for CritFumble, documented in the package description and README.
GameAssist and the submitted source are released under the MIT License.

Project documentation: https://github.com/Mord-Eagle/GameAssist

Detailed release history: https://github.com/Mord-Eagle/GameAssist/blob/main/CHANGELOG.md

### CODE REVIEW

File: GameAssist/GameAssist.js [hunk 1]
```diff
@@ -643,88 +2667,1648 @@
...
+            if (hp < 1) {
+                if (!isDead) requestDeathMarker(token, true);
+                if (hasOpenDeathEntry(token)) return;
+
+                const name = token.get('name') || '(Unnamed NPC)';
+                GameAssist.log('NPCManager', `${name} recorded as dead (HP: ${hp})`);
+
+                // Auto-hide if enabled
+                if (modState.config.autoHide) {
+                    token.set('layer', modState.config.hideLayer);
```
[the remark this answers is not in the document]
Comment (Mord-Eagle) [reply GameAssist/GameAssist.js:4085]: At this point, an NPC on the GM layer is not necessarily considered to be killed or revived by the party and thus is intentionally not tracked.
If I receive requests for such tracking in the future I will look into adding that functionality.

**Comment by Mord-Eagle** [comment]
@codex review

File: GameAssist/GameAssist.js [hunk 2]
```diff
@@ -733,39 +4317,259 @@
...
+                    : '';
+                GameAssist.log('NPCManager', `Configured NPC death marker "${_sanitize(marker)}" could not be recognized during teardown.${detail}`, 'WARN');
+                return;
+            }
+
+            const targets = findObjs({
+                _type: 'graphic',
+                _pageid: pageId,
+                layer: 'objects'
+            }).filter(token => tokenHasMarker(token, resolution.id));
```
[the remark this answers is not in the document]
Comment (Mord-Eagle) [reply GameAssist/GameAssist.js:4409]: @codex address that feedback

[the remark this answers is not in the document]
Comment (Mord-Eagle) [reply GameAssist/GameAssist.js:4409]: This is an edge-case that I do not foresee having issues; but I will test it and update the code if deemed necessary.

**Review by Alicekb — APPROVED** [review]
